jvm/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -132,6 +131,8 @@ private[glutenproject] object GlutenPlugin {
   val SPARK_SESSION_EXTS_KEY: String = StaticSQLConf.SPARK_SESSION_EXTENSIONS.key
   val GLUTEN_SESSION_EXTENSION_NAME: String = Objects.requireNonNull(
     classOf[GlutenSessionExtensions].getCanonicalName)
+  val GLUTEN_WRITE_SESSION_EXTENSION_NAME: String = Objects.requireNonNull(
```

Comment:
Hi, @JkSelf  , please add this extension using OthersExtensionOverrides method, the class 'DwrfWriteExtension' is in velox module, but not in clickhouse module.